tests/cargo-kani/proptest-lib/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! basic proptest examples. First one fails but second does not.
```

Comment:
Isn't it the other way around?

Reply:
Fixed with d25e6d296c9bfb4c0e83171eeb5a6d75666d8b18